zeppelin-server/src/main/java/org/apache/zeppelin/realm/jwt/KnoxJwtRealm.java

```diff
@@ -241,7 +228,6 @@ protected AuthorizationInfo doGetAuthorizationInfo(PrincipalCollection principal
     /* return the groups as seen by Hadoop */
     Set<String> groups = null;
     try {
-      hadoopGroups.refresh();
```

Comment:
Removed this so we could take advantage of hadoop's native group caching mechanisms that should be controllable via hadoop configurations